<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPelanggan</class>
 <widget class="QWidget" name="FormPelanggan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>138</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="kodePelangganLabel">
      <property name="text">
       <string>Kode Pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kodePelangganLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaPelangganLabel">
      <property name="text">
       <string>Nama Pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaPelangganLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="alamatLabel">
      <property name="text">
       <string>Alamat</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="alamatLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="noTelpLabel">
      <property name="text">
       <string>No Telp</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="noTelpLineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
